libraries/libfc/include/fc/variant.hpp

```diff
@@ -694,6 +695,35 @@ namespace fc
       n = boost::multiprecision::number<T>(v.get_string());
    }
 
+   template<typename T> void to_variant( const boost::dynamic_bitset<T>& bs, fc::variant& v ) {
```

Comment:
I'm puzzled why the current implementation stores a `std::vector<fc::variant>` in the variant, and not a `std::vector<T>`?

Reply:
I don't know the original reason behind it.